packages/core/src/dispatcher.ts

```diff
@@ -354,7 +349,7 @@ export default class Dispatcher extends EventEmitter {
 
     clearInterval(this._resubscribeInterval)
 
-    await Promise.all(Object.values(this._documents).map(async (doc) => await doc.close()))
+    await this.repository.close()
```

Comment:
I'm not convinced it makes sense for the Dispatcher to logically "own" the Repository and be responsible for its lifecycle like this.  Kinda feels like this should happen up in `Ceramic.close()`?